docs/CONTRIBUTING.md

```diff
@@ -216,11 +216,11 @@ Must be one of the following:
 - `fix`      Commits that fixes a bug
 - `refactor` Commits that rewrite/restructure your code, however does not change any behaviour
 - `perf`     Commits that are special `refactor` commits that improve performance
-- `style`    Commits that do not affect the meaning (white-space, formatting, missing semi-colons, etc.)
+- `style`    Commits that do not affect the meaning (white-space, formatting, missing semi-colons, etc)

```

Comment:
Same here.

Reply:
Can I update it according to your reviews? 
And then you can merge it.

Reply:
Sure, thanks! Let us know if you need any help. 😄